buildSrc/src/main/groovy/com/uber/okbuck/core/model/JavaLibTarget.groovy

```diff
@@ -50,4 +50,12 @@ class JavaLibTarget extends JavaTarget {
                 project.plugins.hasPlugin('me.tatarka.retrolambda') &&
                 JavaVersion.toVersion(sourceCompatibility) > JavaVersion.VERSION_1_7
     }
+
+    List<Map<String, String>> getTransforms() {
+        return (List<Map<String, String>>) getProp(okbuck.transform.transforms, [])
```

Comment:
transforms only work with android apps, so this should be moved to android app target